<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-Q37MKND" name="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel" tests="8" failures="6" timestamp="06 七月 2020 02:26:41 GMT" time="2.605" errors="0">
  <testcase name="UserInfoQueryData" time="0.115" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;id&quot;:{&quot;currentValue is &quot;:&quot;eef023f445604e7a92f25dd8c1b44c14&quot;,&quot;expectedValue is &quot;:&quot;782783112a42461298fbea5e0b5942c3&quot;},&quot;nickName&quot;:{&quot;currentValue is &quot;:&quot;SaaS&quot;,&quot;expectedValue is &quot;:&quot;新源科技&quot;},&quot;title&quot;:{&quot;currentValue is &quot;:&quot;&quot;,&quot;expectedValue is &quot;:&quot;Boss&quot;},&quot;createTime&quot;:{&quot;currentValue is &quot;:&quot;2019-07-05 11:03:31&quot;,&quot;expectedValue is &quot;:&quot;2019-07-12 15:18:48&quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"id":{"currentValue is ":"eef023f445604e7a92f25dd8c1b44c14","expectedValue is ":"782783112a42461298fbea5e0b5942c3"},"nickName":{"currentValue is ":"SaaS","expectedValue is ":"新源科技"},"title":{"currentValue is ":"","expectedValue is ":"Boss"},"createTime":{"currentValue is ":"2019-07-05 11:03:31","expectedValue is ":"2019-07-12 15:18:48"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.182" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:3,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:3,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:3,&quot;expectLenth&quot;:1},&quot;id&quot;:{&quot;currentValue is &quot;:&quot;d2d5817f9a59463ebd0ef841d9820bad&quot;,&quot;expectedValue is &quot;:&quot;c2da7bb67b354e32b140fa4676310de1&quot;},&quot;username&quot;:{&quot;currentValue is &quot;:&quot;hangang3@bitnei.cn&quot;,&quot;expectedValue is &quot;:&quot;hangang@bitnei.cn&quot;},&quot;nickName&quot;:{&quot;currentValue is &quot;:&quot;hangang三&quot;,&quot;expectedValue is &quot;:&quot;hangang&quot;},&quot;email&quot;:{&quot;currentValue is &quot;:&quot;hangang3@bitnei.cn&quot;,&quot;expectedValue is &quot;:&quot;hangang@bitnei.cn&quot;},&quot;mobile&quot;:{&quot;currentValue is &quot;:&quot;131****3331&quot;,&quot;expectedValue is &quot;:&quot;189****6589&quot;},&quot;title&quot;:{&quot;currentValue is &quot;:&quot;tste&quot;,&quot;expectedValue is &quot;:&quot;&quot;},&quot;createBy&quot;:{&quot;currentValue is &quot;:&quot;SaaS@bitnei.cn&quot;,&quot;expectedValue is &quot;:&quot;hangang@bitnei.cn&quot;},&quot;createTime&quot;:{&quot;currentValue is &quot;:&quot;2020-06-15 16:46:21&quot;,&quot;expectedValue is &quot;:&quot;2019-09-16 13:50:38&quot;},&quot;origin&quot;:{&quot;currentValue is &quot;:&quot;ADD&quot;,&quot;expectedValue is &quot;:&quot;APPLY&quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":3,"expectedValue is":1},"total":{"currentValue is ":3,"expectedValue is":1},"Length":{"actualLenth":3,"expectLenth":1},"id":{"currentValue is ":"d2d5817f9a59463ebd0ef841d9820bad","expectedValue is ":"c2da7bb67b354e32b140fa4676310de1"},"username":{"currentValue is ":"hangang3@bitnei.cn","expectedValue is ":"hangang@bitnei.cn"},"nickName":{"currentValue is ":"hangang三","expectedValue is ":"hangang"},"email":{"currentValue is ":"hangang3@bitnei.cn","expectedValue is ":"hangang@bitnei.cn"},"mobile":{"currentValue is ":"131****3331","expectedValue is ":"189****6589"},"title":{"currentValue is ":"tste","expectedValue is ":""},"createBy":{"currentValue is ":"SaaS@bitnei.cn","expectedValue is ":"hangang@bitnei.cn"},"createTime":{"currentValue is ":"2020-06-15 16:46:21","expectedValue is ":"2019-09-16 13:50:38"},"origin":{"currentValue is ":"ADD","expectedValue is ":"APPLY"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.141" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;id&quot;:{&quot;currentValue is &quot;:&quot;eef023f445604e7a92f25dd8c1b44c14&quot;,&quot;expectedValue is &quot;:&quot;782783112a42461298fbea5e0b5942c3&quot;},&quot;nickName&quot;:{&quot;currentValue is &quot;:&quot;SaaS&quot;,&quot;expectedValue is &quot;:&quot;新源科技&quot;},&quot;title&quot;:{&quot;currentValue is &quot;:&quot;&quot;,&quot;expectedValue is &quot;:&quot;Boss&quot;},&quot;createTime&quot;:{&quot;currentValue is &quot;:&quot;2019-07-05 11:03:31&quot;,&quot;expectedValue is &quot;:&quot;2019-07-12 15:18:48&quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"id":{"currentValue is ":"eef023f445604e7a92f25dd8c1b44c14","expectedValue is ":"782783112a42461298fbea5e0b5942c3"},"nickName":{"currentValue is ":"SaaS","expectedValue is ":"新源科技"},"title":{"currentValue is ":"","expectedValue is ":"Boss"},"createTime":{"currentValue is ":"2019-07-05 11:03:31","expectedValue is ":"2019-07-12 15:18:48"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="1.683" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
  <testcase name="UserInfoQueryData" time="0.169" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;id&quot;:{&quot;currentValue is &quot;:&quot;eef023f445604e7a92f25dd8c1b44c14&quot;,&quot;expectedValue is &quot;:&quot;782783112a42461298fbea5e0b5942c3&quot;},&quot;nickName&quot;:{&quot;currentValue is &quot;:&quot;SaaS&quot;,&quot;expectedValue is &quot;:&quot;新源科技&quot;},&quot;title&quot;:{&quot;currentValue is &quot;:&quot;&quot;,&quot;expectedValue is &quot;:&quot;Boss&quot;},&quot;createTime&quot;:{&quot;currentValue is &quot;:&quot;2019-07-05 11:03:31&quot;,&quot;expectedValue is &quot;:&quot;2019-07-12 15:18:48&quot;}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"id":{"currentValue is ":"eef023f445604e7a92f25dd8c1b44c14","expectedValue is ":"782783112a42461298fbea5e0b5942c3"},"nickName":{"currentValue is ":"SaaS","expectedValue is ":"新源科技"},"title":{"currentValue is ":"","expectedValue is ":"Boss"},"createTime":{"currentValue is ":"2019-07-05 11:03:31","expectedValue is ":"2019-07-12 15:18:48"}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.091" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.106" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel">
    <failure type="java.lang.AssertionError" message="expected [{}] but found [{&quot;size&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;pages&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;total&quot;:{&quot;currentValue is &quot;:0,&quot;expectedValue is&quot;:1},&quot;Length&quot;:{&quot;actualLenth&quot;:0,&quot;expectLenth&quot;:1}}]">
      <![CDATA[java.lang.AssertionError: expected [{}] but found [{"size":{"currentValue is ":0,"expectedValue is":1},"pages":{"currentValue is ":0,"expectedValue is":1},"total":{"currentValue is ":0,"expectedValue is":1},"Length":{"actualLenth":0,"expectLenth":1}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel.UserInfoQueryData(UserInfoQueryByExcel.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- UserInfoQueryData -->
  <testcase name="UserInfoQueryData" time="0.118" classname="com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel"/>
</testsuite> <!-- com.bitnei.apitest.testcases.saas.UserInfoQueryByExcel -->
